src/CMakeLists.txt

```diff
@@ -7,7 +7,10 @@ set(SOURCES
 
 	TriMesh.h
 	TriMesh.cpp
-	Tuple.cpp
+
+	EdgeCollapsingConn.cpp
```

Comment:
What does `Conn` mean? Connected? Connectivity? Co-Neural-Network? Conegative?

Reply:
`Connectivity`, This is a merge patch from main.

Reply:
We can discuss the naming rule togethehr.

Reply:
Sorry I just have a tendency to pickup on abbreviations, which I think should be avoided :D